### PR TITLE
Python 310

### PR DESCRIPTION
**Pull request by hoffmang9**
Attempting to get Python 3.10 working across the application.

The challenge is that 3.10 deprecated loop as a the loop parameter is deprecated from all APIs in 3.10.

We may have to go back to removing websockets as it is only used in daemon.

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -123,14 +123,14 @@ fi
...
+	for V in 310 3.10 39 3.9 38 3.8 37 3.7 3; do
 		if command -v python$V >/dev/null; then
 			if [ "$BEST_VERSION" = "" ]; then
 				BEST_VERSION=$V
 				if [ "$BEST_VERSION" = "3" ]; then
 					PY3_VERSION=$(python$BEST_VERSION --version | cut -d ' ' -f2)
-					if [[ "$PY3_VERSION" =~ 3.10.* ]]; then
-						echo "Chia requires Python version <= 3.9.9"
+					if [[ "$PY3_VERSION" =~ 3.11.* ]]; then
+						echo "Chia requires Python version <= 3.11.0"
```
Comment (hoffmang9) [inline install.sh:133]: Need to not forget to fix this to less than

**Comment by mariano54** [comment]
You might need this PR as well:
https://github.com/Chia-Network/chia-blockchain/pull/9736

Due to  `__annotations__` changes on python 3.10.

**Comment by hoffmang9** [comment]
Closing in favor of #9808

**Comment by altendky** [comment]
https://github.com/Chia-Network/chia-blockchain/pull/9808 just updates deps.  Isn't the rest of this (and more) needed?

**Comment by hoffmang9** [comment]
A whole lot more is needed as my initial pass still has failing Daemon. I'm just not going to have time to take it further soon. Branch in question is not deleted.

**Comment by altendky** [comment]
I duplicated and catch-up merged this branch over in https://github.com/Chia-Network/chia-blockchain/pull/9930.  I also added some brief explanation of how I see the effort moving forward with multiple people fixing a piece at a time.

**Pull request closed without merging**